src/pages/costModels/costModel/updateMarkupDialog.tsx

```diff
@@ -32,23 +38,52 @@ interface Props extends WithTranslation {
 
 interface State {
   markup: string;
+  origIsDiscount: boolean;
+  isDiscount: boolean;
 }
 
 class UpdateMarkupModelBase extends React.Component<Props, State> {
   constructor(props) {
     super(props);
+    const initialMarkup = this.props.current.markup.value;
+    const isNegative = Number(initialMarkup) < 0;
+    const noSignValue = isNegative ? initialMarkup.substring(1) : initialMarkup;
```

Comment:
Should this have a default of zero? I ask because there is a zero default below on line 10.

    `const noSignValue = isNegative ? initialMarkup.substring(1) : initialMarkup || 0;`